wiki/Guides/Tablet_Purchase/en.md

```diff
@@ -1,54 +1,74 @@
-**In short:** Tablets are an optional device to let you play the game in a different way. They were originally intended for digital art, but happen to work very well with osu!, and most specifications will make no difference in osu!.
 
-Types of Tablets
-----------------
+The biggest selling point of tablets for osu! is this absolute tracking. Every other parameter such as pressure levels, tilt sensitivity, barrel rotation, etc. will have no effect on your osu! gameplay. These features are for digital art, where the tablet can sense things such as the pressure you're applying to make a darker pencil mark, for example. The only specification which may affect your osu! gameplay is LPI (lines per inch). This is essentially the "resolution" of the tablet surface, or how little you have to move for the tablet to detect it. If a tablet has 100 LPI, then you have to move at least 1/100th of an inch for it to register movement. However, even the lowest LPI should be far higher than any monitor that you're likely to play on, so higher numbers have a mostly unnoticeable effect on gameplay.
+
+tl;dr Tablets are an optional device to let you play the game in a different way. They were originally intended for digital art, but happen to work very well when playing osu!. Most specifications will make no difference in osu!.
+
+## Tablet Vendors
+
+### osu!store
+
+*Notice: **The osu!store no longer sells osu!tablets.*** At this time, [peppy has twitted](https://twitter.com/ppy/status/846190638873767936) that if you want to try this input method, you may want to look into other alternatives.
+
+With the goal of trying to allow players to try the tablet input method without putting a hole in their wallet, the osu!store had featured the osu!tablet. It was made by Huion and designed by flyte. The first version in 2013 featured the osu!tablet (white), a pen with a nib, 3 replaceable nibs, but required a AA battery for the pen (which was included but would add weight to the pen). The second version in 2016 feature the osu!tablet (black), a pen with a nib, 3 replaceable nibs, but the pen needed to be charged via USB (included).
+
+![](img/osu!tabletv1.jpg "osu!tablet v1 \(2013\)")
+
+![](img/osu!tabletv2.jpg "osu!tablet v2 \(2016\)")
+
+### Huion
+
+Huion is a tablet company that is the supplier of the osu!tablet. They are best own to be cheaper than Wacom tablets by a large margin but some would say that their tablets feel flimsier and that running a tablet pen over a Huion tablet would feel as if you were on writing paper (in comparison, some say that on Wacom, it is as if you were writing on glass). Their tablet pens either require a battery or needs to be charged, unlike Wacom tablet pens, and their nibs may need to be often than Wacom nibs. Other than that, there really are not any noticeable differences between Walcom and Huion tablets.
```

Comment:
i have no idea what 'best own to be cheaper' is meant to actually be

this whole paragraph also doesn't really say a lot outside of circumstantial hearsay

Reply:
whoops, I thought that was a typo, it was supposed to read:
`They are best known for being cheaper`

I wasn't sure about what else to write about Huion tablets, I was sorta basing this off of another forum and a youtube video